activation/handler_test.go

```diff
@@ -679,38 +674,12 @@ func TestHandler_SyntacticallyValidateAtx(t *testing.T) {
 		require.Nil(t, proof)
 	})
 
-	t.Run("missing NodeID in initial atx", func(t *testing.T) {
-		t.Parallel()
-
-		atxHdlr := newTestHandler(t, goldenATXID)
-
-		ctxID := posAtx.ID()
-		challenge := types.NIPostChallenge{
-			Sequence:       0,
-			PrevATXID:      types.EmptyATXID,
-			PublishEpoch:   currentLayer.GetEpoch(),
-			PositioningATX: posAtx.ID(),
-			CommitmentATX:  &ctxID,
-		}
-		nipost := types.NIPost{PostMetadata: &types.PostMetadata{}}
-		atx := newAtx(challenge, &nipost, 100, types.GenerateAddress([]byte("aaaa")))
-		atx.InitialPost = &types.Post{
-			Nonce:   0,
-			Indices: make([]byte, 10),
-		}
-
-		atxHdlr.mclock.EXPECT().CurrentLayer().Return(currentLayer)
-		err := atxHdlr.SyntacticallyValidate(context.Background(), atx)
-		require.ErrorContains(t, err, "node id is missing")
-	})
```

Comment:
This test should not be deleted. NodeID must be included in the initial ATX!

Reply:
This case is covered now by a different test.